packages/ra-core/src/util/DependsOn.js

```diff
@@ -47,7 +48,7 @@ export const DependentInputComponent = ({
     );
 };
 
-DependentInputComponent.propTypes = {
+DependsOnView.propTypes = {
```

Comment:
Why `DependsOnView` instead of `DependsOn`?

Reply:
Because of the default export which is `DependsOn`. That will avoid eslint warning about importing a named export as default